lib/credit_card_form.dart

```diff
@@ -247,7 +248,8 @@ class _CreditCardFormState extends State<CreditCardForm> {
                           final List<String> date = value.split(RegExp(r'/'));
                           final int month = int.parse(date.first);
                           final int year = int.parse('20${date.last}');
-                          final DateTime cardDate = DateTime(year, month);
+                          final int lastDayofMonth = DateTime(year, Jiffy( DateTime(year,month)).add(months: 1).month, 0 ).day;
```

Comment:
@ocakliemre can we use another way to get the last date of the month, as we do not want any additional package dependencies?